src/index.stories.js

```diff
@@ -1,6 +1,50 @@
 import React from 'react';
 import { storiesOf } from '@kadira/react-native-storybook';
+// import { Router, Route, Actions, routerReducer } from 'react-native-redux-router';
+import { Scene, Router, Actions } from 'react-native-router-flux';
+import { Button, View } from 'react-native';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
 
+import reducers from './reducers';
+import ExerciseApp from './containers/ExerciseApp';
+import Splash from './components/exercises/Splash';
 import App from '.';
 
+// default
 storiesOf('App', module).add('default', () => <App />);
+
+// router demonstration
+const reducer = combineReducers(reducers);
+const store = createStore(reducer);
```

Comment:
I think this should be done in the story itself, otherwise the store is not reset when one switches between stories (and then you wonder why the forward button does not work)